src/test/java/uk/ac/ebi/eva/pipeline/jobs/AnnotationJobTest.java

```diff
@@ -83,7 +87,19 @@
     public void allAnnotationStepsShouldBeExecuted() throws Exception {
         mongoRule.restoreDump(getResourceUrl(MONGO_DUMP), jobOptions.getDbName());
 
-        JobExecution jobExecution = jobLauncherTestUtils.launchJob();
+        JobParameters jobParameters = new EvaJobParameterBuilder()
+                .vepPath(getResource(MOCK_VEP).getPath())
+                .vepCacheVersion("")
+                .vepCachePath("")
+                .vepCacheSpecies("")
+                .inputFasta("")
+                .vepNumForks("")
+                .outputDirAnnotation(OUTPUT_DIR_ANNOTATION)
+                .inputStudyId(INPUT_STUDY_ID)
+                .inputVcfId(INPUT_VCF_ID)
+                .toJobParameters();
+
+        JobExecution jobExecution = jobLauncherTestUtils.launchJob(jobParameters);
 
         assertEquals(ExitStatus.COMPLETED, jobExecution.getExitStatus());
         assertEquals(BatchStatus.COMPLETED, jobExecution.getStatus());
```

Comment:
Well in the near future all these parameters will be validated at the beginning so if we reach this point we are sure that they are valid... and not empty ;)